sretoolbox/container/image.py

```diff
@@ -35,8 +35,11 @@ class Image:
                          the tag provided in the url or the default one
     :param username: (optional) The private registry username
     :param password: (optional) The private registry password
+    :param auth_server: (optional) The host that the username and password are
+                        meant for
     """
-    def __init__(self, url, tag_override=None, username=None, password=None):
+    def __init__(self, url, tag_override=None, username=None, password=None,
+                 auth_server=None):
```

Comment:
thinking about this a bit, wouldn't the server be included in the url?

Reply:
aren't you passing credentials meant for one registry to another?

Reply:
you are right. somehow i thought that the auth server is also defined in the credentials (thinking about a dockercfg, which is not the case). please ignore :)